src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
```

Comment:
>System [](http://example.com/codeflow?start=6&length=6)

Is this using necessary?

Reply:
Yes, for Array.Empty for conditional diagnostics in one test